examples/grpc-bridge/config/s2s-python-envoy.json

```diff
@@ -0,0 +1,74 @@
+{
+  "listeners": [
+    {
+      "port": 9001,
+      "filters": [
+        {
+          "type": "read",
+          "name": "http_connection_manager",
+          "config": {
+            "codec_type": "auto",
+            "add_user_agent": true,
+            "idle_timeout_s": 840,
+            "access_log": [
+              {
+                "path": "/var/log/envoy/egress_http_error.log",
```

Comment:
i'd just make this file egress_http.log as there is nothing about it to be error (no filters applied)
or remove access_log configuration entierly as it's not important for demo but keep the config succinct.